weldx/tests/test_utility.py

```diff
@@ -498,3 +499,125 @@ def test_xr_time_ref():
 
     da2.weldx.time_ref = t0
     assert da2.time.attrs["time_ref"] == t0
+
+
+class TestCompareNested:
+    @staticmethod
+    @pytest.fixture()
+    def _default_dicts():
+        """Return two equivalent deeply nested structures to be modified by tests."""
+        a = {
+            "foo": np.arange(3),
+            "x": {
+                0: [1, 2, 3],
+            },
+            "bar": True,
+        }
+        b = copy.deepcopy(a)
+        return a, b
+
+    @pytest.mark.parametrize(["a", "b"], [("asdf", "foo"), (b"asdf", b"foo"), (1, 2)])
```

Comment:
Might be picky and a matter of taste, but I like one line per test case. If the auto-formatting is an issue, just put a `,` behind the last tuple

Reply:
yes, this has been formatted by black. Thanks for the comma hint ;)